functions/venue.js

```diff
@@ -443,6 +448,10 @@ exports.updateVenue = functions.https.onCall(async (data, context) => {
     updated.showRangers = data.showRangers;
   }
 
+  if (typeof data.showReactions === "boolean") {
+    updated.showReactions = data.showReactions;
+  }
```

Comment:
You could do:

```
if (data.showReactions) {
  updated.showReactions = isTruthy(data.showReactions);
}
```

Reply:
This approach works only if `data.showReactions` is true, if it is false, the `updated` will not get the value